docs/sdk-docs/private-chain/README.md

```diff
@@ -0,0 +1,144 @@
+
+ 
+## Setting up the Development Environment
+With the commit or tag of a compatible python client, you can go through the [tutorial] to set up your private network.
+
+Assuming the `develop` branch uses a newer contract version, you might need to slightly deviate from the tutorial. More specific you need to check out the correct branch or tag after [cloning] Raiden.
+
+For example, if you need to you the `v0.200.0-rc2` tag, you need to first checkout the tag before continuing with the tutorial.
+
+```bash
+git checkout v0.200.0-rc2
+```
+
+At the end of the guide you should have `Geth` running a private chain, with the Raiden contracts deployed, and a Raiden node running.
+
+### Locating the Smart contracts
+Now you bootstrapped the network, and you have to locate the deployment files.
+
+The deployment files will point the SDK to the correct smart contracts.
+
+Look for `env/lib/python3.7/site-packages/raiden_contracts/data_${VERSION}/`, in the directory that contains your virtual environment *(the one you created while following the tutorial)*.
+
+You should find the two following files in this directory:
+
+ * `deployment_private_net.json`
+ * `deployment_services_private_net.json`
+
+Copy these files to your project's `src/deployment` and proceed with the SDK or dApp initialization.
+
+### Starting the SDK
+You can start using the SDK in your setup.
+
+To start the SDK on your private network setup, you need to import the private deployment files. You can find them in the `<rootDir>/src/deployment` directory.
+
+```typescript
+import privateDeployment from '@/deployment/deployment_private_net.json';
+import privateServicesDeployment from '@/deployment/deployment_services_private_net.json';
+import { ContractsInfo } from 'raiden-ts';
+```
+
+As soon as the imports exist, you need to construct the `contractsInfo` object and pass it to the Light Client SDK initialization logic. 
+
+An example of the SDK initialization function follows:
+
+```typescript
+async function initialization() {
+    const contractsInfo = ({
+      ...privateDeployment.contracts,
+      ...privateServicesDeployment.contracts
+    } as unknown) as ContractsInfo;
+
+    const raiden = await Raiden.create(
+        provider,
+        0,
+        window.localStorage,
+        contractsInfo
+    );
+}
+```
+
+Calling the `initialization()` function should enable the SDK to manage channels and send payments in the development environment, using the previously deployed token.
+
+### Changing the dApp to work on the private setup
+Out of the box the dApp supports the following public test networks `Görli`, `Rinkeby` and `Ropsten`.
+
+To run the dApp on any other network setup you need to deploy the smart contracts and change the dApp to use this setup.
+
+If you require help running the dApp, you can follow the [instructions].
+
+As soon as you have the dApp running you can locate the [`createRaiden`] function and change it accordingly. 
```

Comment:
We can specify that it is in the `raiden-service.ts`.

Reply:
Doesn't following the link provide that context?

Reply:
Also, the instructions link helps with cloning the dApp.

Reply:
Yes it does, I did not click it so I missed that it linked to the source file. Very good.

Reply:
Can we resolve this? or do you still think that it makes sense to specify the file?

Reply:
No it is good the way it is so we can resolve it. :)